### PR TITLE
Show block previews in levelbuilder

### PR DESCRIPTION
**Pull request by Hamms**
It's not perfect; specifically, it only works on _update_ rather than on _initialization_, because it's not clear to me how to hook into the "preview area fully rendered" event. Still, it's a lot better than nothing!

### Instructions

Before | After
--- | ---
image | image

### Authored Hints

Before | After
--- | ---
image | image

### CODE REVIEW

File: apps/src/sites/studio/pages/levelbuilder.js [hunk 1]
```diff
@@ -36,3 +37,4 @@ window.CodeMirror = codemirror;
 // TODO: Remove when global `marked` is no longer required.
 window.marked = marked;
 window.renderer = renderer;
+window.convertXmlToBlockly = convertXmlToBlockly;
```
Comment (joshlory) [inline apps/src/sites/studio/pages/levelbuilder.js:42]: Is this a sign the JS in `_authored_hints.haml` haml below should be extracted to a JS file?

**Review by joshlory — APPROVED** [review]
Would love to see the js-in-haml cleaned up, but not blocking.

File: apps/src/sites/studio/pages/levelbuilder.js (hunk 1 above)
Comment (Hamms) [reply apps/src/sites/studio/pages/levelbuilder.js:42]: yeah, most definitely. I'll add a TODO

Comment (joshlory) [reply apps/src/sites/studio/pages/levelbuilder.js:42]: Or better yet, a Jira task?

Comment (Hamms) [reply apps/src/sites/studio/pages/levelbuilder.js:42]: :+1: https://codedotorg.atlassian.net/secure/RapidBoard.jspa?rapidView=21&projectKey=LP&view=planning&selectedIssue=LP-150

**Pull request merged**